bridge/src/lib.rs

```diff
@@ -10,12 +10,12 @@ mod worker;
 use connection::{Connection, RpcParams};
 use runtime::Runtime;
 use rutie::{
-    Module, Object, Symbol, RString, Encoding, AnyObject, AnyException, Exception, VM, Thread,
-    NilClass, Hash, Integer, Boolean, Array,
+    AnyException, AnyObject, Array, Boolean, Encoding, Exception, Hash, Integer, Module, NilClass,
+    Object, RString, Symbol, Thread, VM,
 };
 use std::collections::HashMap;
 use temporal_sdk_core_api::telemetry::{Logger, TelemetryOptionsBuilder};
-use test_server::{TestServer, TestServerConfig, TemporaliteConfig};
+use test_server::{TemporaliteConfig, TestServer, TestServerConfig};
```

Comment:
nit: while you're here change to use our new CLI.
Doesn't have to be in this PR.

Reply:
Yes, I'm getting to this soon.